Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/JoinSessionMessage.java

```diff
@@ -18,6 +20,11 @@
     private final int capacity;
     private final GameSettings settings;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)